recipes/selectolax/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "selectolax" %}
+{% set version = "0.3.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/selectolax-{{ version }}.tar.gz
+  sha256: d71d7ae7bec6e856b83111f270bc783e531d86e02ad3a78a37d8591ff8f3620f
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython >=0.29.23
+    - pip
+    - python
+  run:
+    - python
+    #- cython >=0.29.23   
+
+test:
+  imports:
+    - selectolax
+  commands:
+    - pip check
+  requires:
+    - pip
+    - cython >=0.29.23
```

Comment:
Can you add a comment here that this is needed to make `pip check` pass even though cython isn't *actually* needed at runtime?

Reply:
Done.

Reply:
Thanks!

We still need to check [this](https://github.com/conda-forge/staged-recipes/pull/17395#issuecomment-1004673878).